docs/developer-docs/docs/operate-a-node/node-commands.md

```diff
@@ -0,0 +1,145 @@
+﻿---
+sidebar_position: 7
+---
+
+# Node commands
+
+## Overview
+
+**Node commands** are commands for interacting with a node through the command line. In this article, you'll find a brief overview of commands for interacting with the Warden node, `wardend`.
+
+These commands allow you to do the following:
+
+- Query the node
+- Initiate transactions
+- Manage your key
+- Edit the genesis file
+- and much more
+
+## Execute `wardend` commands
+
+To execute a node command, just run `wardend`, followed by the command.
+
+For example, to check the node status, run this:
+
+```bash
+wardend status
```

Comment:
`wardend status` will not work unless a chain is running or connected to chain via `--node` flag